core/common/assets/js/utils/environment.js

```diff
@@ -33,7 +33,12 @@ const matchUserAgent = ( UserAgentStr ) => {
 	// Apple Webkit engine
 	isAppleWebkit = matchUserAgent( 'AppleWebKit' ) && ! isBlink,
 
+	isTouchDevice = ( 'ontouchstart' in window ) ||
+		( navigator.maxTouchPoints > 0 ) ||
+		( navigator.msMaxTouchPoints > 0 ),
+
```

Comment:
@TzviRabinovitch For sanity, please just double check for me if this is returning the correct value?
When I run the old way I get `false` for my Desktop view, but with this method I get `true`.
Check the attached screenshot on how I tested it.
![Screenshot 2023-01-23 at 16 33 25](https://user-images.githubusercontent.com/20996103/214066307-0b4ae03d-4a9a-47fb-87a9-e0d9cc510294.png)

Reply:
Okay cool.